lib/MusicBrainz/Server/EditSearch/Predicate/EditorFlag.pm

```diff
@@ -1,19 +1,44 @@
 package MusicBrainz::Server::EditSearch::Predicate::EditorFlag;
 use Moose;
+use MusicBrainz::Server::Constants qw( $UNTRUSTED_FLAG );
 use namespace::autoclean;
 
 extends 'MusicBrainz::Server::EditSearch::Predicate::Set';
 
+has user => (
+    isa => 'Editor',
+    is => 'ro',
+    required => 1,
+);
+
+sub valid_flags {
+    my ($self) = @_;
+
+    my @flags = @{ $self->sql_arguments->[0] };
+    unless ($self->user->is_admin) {
```

Comment:
If it is to be restricted, I don’t think it makes sense to show it to wiki transcluders, relationship editors etc. This should be `is_account_admin` instead (everywhere else in this PR, too).

(And unrelatedly, `is_admin` should probably be extended to test for account admins, too.)